NetKAN/Kopernicus.netkan

```diff
@@ -3,9 +3,13 @@
     "identifier"    : "Kopernicus",
     "name"          : "Kopernicus Planetary System Modifier",
     "abstract"      : "Allows users to replace the planetary system used by the game.",
-    "$kref"         : "#/ckan/github/Kopernicus/Kopernicus",
+    "$kref"         : "#/ckan/github/Kopernicus/Kopernicus/version_from_asset/^Kopernicus-(?<version>.+).zip$",
```

Comment:
```suggestion
    "$kref"         : "#/ckan/github/Kopernicus/Kopernicus/version_from_asset/^Kopernicus-(?<version>.+)\\.zip$",
```

I think this is the syntax for matching a period in a regex from JSON.

Reply:
Whoops, thanks. Had it in my test .netkan for the BE fork, but of course had to forget it here :roll_eyes: